### PR TITLE
#77 Add token accounts to AccountMeta

### PR DESCRIPTION
**Pull request by anton-lisanin**
Add token accounts to AccountMeta

### CODE REVIEW

File: proxy/plugin/solana_rest_api_tools.py [hunk 1]
```diff
@@ -642,7 +657,12 @@ def createEtherAccountTrx(client, ether, evm_loader_id, signer, code_acc=None):
             keys=[
                 AccountMeta(pubkey=base, is_signer=True, is_writable=True),
                 AccountMeta(pubkey=PublicKey(sol), is_signer=False, is_writable=True),
+                AccountMeta(pubkey=associated_token, is_signer=False, is_writable=True),
                 AccountMeta(pubkey=system, is_signer=False, is_writable=False),
+                AccountMeta(pubkey=ETH_TOKEN_MINT_ID, is_signer=False, is_writable=False),
+                AccountMeta(pubkey=TOKEN_PROGRAM_ID, is_signer=False, is_writable=False),
+                AccountMeta(pubkey=ASSOCIATED_TOKEN_PROGRAM_ID, is_signer=False, is_writable=False),
```
Comment (s-medvedev) [inline proxy/plugin/solana_rest_api_tools.py:666]: Do you need to add this account?

**Review by s-medvedev — APPROVED** [review]

File: proxy/plugin/solana_rest_api_tools.py [hunk 2]
```diff
@@ -40,6 +43,11 @@
 rentid = "SysvarRent111111111111111111111111111111111"
 system = "11111111111111111111111111111111"
 
+ETH_TOKEN_MINT_ID: PublicKey = PublicKey(
+    os.environ.get("ETH_TOKEN_MINT", "HPsV9Deocecw3GeZv1FkAPNCBRfuVyfw9MMwjwRe1xaU")
```
Comment (s-medvedev) [inline proxy/plugin/solana_rest_api_tools.py:47]: How can someone create a mint with this public key?

**Review by s-medvedev — DISMISSED** [review]

**Review by s-medvedev — COMMENTED** [review]
Looks good. But we will merge all PR together.

File: proxy/plugin/solana_rest_api_tools.py (hunk 1 above)
Comment (anton-lisanin) [reply proxy/plugin/solana_rest_api_tools.py:666]: Yes, I need this

**Comment by otselnik** [comment]
`proxy/solana-py.patch` have no need to be changed and does not work with this changes

**Review by vakond — DISMISSED** [review]
Studied this.

**Review by s-medvedev — APPROVED** [review]

**Pull request merged**